app/models/setting.py

```diff
@@ -63,6 +63,9 @@ class Setting(db.Model):
     # Instagram
     in_client_id = db.Column(db.String)
     in_client_secret = db.Column(db.String)
+    # Github
+    git_client_id = db.Column(db.String)
+    git_client_secret = db.Column(db.String)
```

Comment:
Please change to `github_client_id`

git != github

Reply:
Doing so